app/celery/scheduled_tasks.py

```diff
@@ -153,25 +154,36 @@ def _check_slow_text_message_delivery_reports_and_raise_error_if_needed(reports:
     # If over 10% of all text messages sent over the period have taken longer than 5 minutes to deliver, let's flag a
     # sentry error for us to investigate.
     if percent_slow_notifications >= 10:
-        with sentry_sdk.push_scope() as scope:
-            error_context = {
-                "Support runbook": (
-                    "https://github.com/alphagov/notifications-manuals/wiki/Support-Runbook#slow-sms-delivery"
-                ),
-                "Slow text messages - #": slow_notifications,
-                "Slow text messages - %": percent_slow_notifications,
-                "Total text messages": total_notifications,
-            }
+        count = redis_store.incr(CacheKeys.NUMBER_OF_TIMES_OVER_SLOW_SMS_DELIVERY_THRESHOLD)
+
+        # If this is the third consecutive time we've seen the threshold breached, then we log an error to Sentry.
+        # This tells us that for at least 15 minutes we've seen delivery take longer than 5 minutes for >10% of
+        # requests. At that point it's probably worth investigating. By only checking for == 3, we don't log any
```

Comment:
```suggestion
        # If this is the fifth consecutive time we've seen the threshold breached, then we log an error to Sentry.
        # This tells us that for at least 15 minutes we've seen delivery take longer than 5 minutes for >10% of
        # requests. At that point it's probably worth investigating. By only checking for == 5, we don't log any
```

i think the sentry msg is obvious enough (given that the first action i'd probably take after seeing this would be to check grafana and see the provider graphs) but if we do want to capture the nuance that @whpearson raised around it reflecting ~20 minutes then this comment is a reasonable spot